### DISCUSSION TITLE
안녕하세요! 프로젝트를 응원하고 싶습니다.

### DISCUSSION THREAD

**Discussion by RikanDev** [dlunch/wie#284] [General]
안녕하세요! 국산 피처폰 게임은 돌릴 수 있는 에뮬이 없어 안타까움을 느끼고 있던 와중에 거의 유일하다시피 WIPI 에뮬 개발에 도전 중이신 분을 발견하여 정말 너무 감사하고 반갑습니다! 제가 프로그래밍 지식이 미달한 관계로 기술적인 도움은 드리지 못하고 그저 멀리서 프로젝트를 응원해 드리는 것밖에 할 수가 없어 이렇게라도 이슈탭에 글을 남겨봅니다. (깃허브는 처음 사용해보기에 제가 잘못 사용하고 있다면 죄송합니다.)

혹시 오디오 구현은 어느 정도 진행되셨는지 여쭈어도 될까요? 아직 많은 게임은 돌릴 수 없지만 오디오만 구현되어도 미니게임천국이나 리듬스타만 해도 추억하는 사람이 많으니 다시 플레이해볼 수 있다는 것에 어느 정도 화제가 될 수 있고 저도 여러 군데 이 프로젝트를 알리는데 자그마한 도움을 드릴 수도 있으니 프로젝트가 좀 더 많은 관심과 후원, 기여자를 모을 수 있지 않을까 싶어 조심스레 여쭤봅니다.

그 외 제가 프로젝트를 지지하고 도와드릴 방법이 없을까 생각해보았는데 게임별로 일일이 실행 여부나 오류 메시지 등을 정리해본 표를 만들어볼까 생각 중입니다만 이러한 작업이 도움이 될 수 있을까요?

**Comment by dlunch** [comment]
안녕하세요, 관심 가져주셔서 감사합니다!

오디오 구현은 현재 어느정도 진행이 된 상태구요(https://github.com/dlunch/smaf), 현재 연동 작업을 진행해야 하는 상태입니다. 조만간 마무리가 되지 않을까 싶네요.

게임별 호환성 테이블은 https://github.com/dlunch/wie/wiki/Compatibility 여기에 제가 따로 정리해 두고 있어 따로 마련해 주지 않으셔도 될 것 같습니다. 여기에 없는 앱들은 따로 요청해 주시면 제가 테스트해보고 작성하는 방식으로 하려고 합니다. 아직 에러 처리가 잘 돼있지 않아 어떤 부분에서 문제가 되는지 알기 어려운 상태여서 제가 확인하는게 현 상태에서는 나을 것 같습니다.

감사합니다!

**Comment by RikanDev** [comment]
넵 정말 감사합니다! 앞으로도 계속 응원하겠습니다 :D
